packages/runtime/src/useSSRComputation_Server.tsx

```diff
@@ -9,12 +10,13 @@ export default function useSSRComputation_Server(fn: (...dependencies: any[]) =>
     // relativePathToCwd is used to make sure that the cache key is unique for each module
     // and it's not affected by the file that calls it
     const cacheKey = calculateCacheKey(relativePathToCwd, dependencies);
-    // check if result is a promise
-    if (result && typeof result.then === 'function') {
-      cache[cacheKey] = result.then(asyncResult => cache[cacheKey] = asyncResult);
-    } else {
-      cache[cacheKey] = result;
+
+    // check if it should ignore caching
+    if (result && result[IgnoreCache]) {
```

Comment:
If we want this feature, this has to be duplicated client-side, as it will return different stuff client-side and server-side, resulting in crashes and hydration conflicts.